clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/PalisadeRecordReaderTest.java

```diff
@@ -142,19 +142,19 @@ public void shouldReturnFalseAfterClosed() throws IOException {
      *
      * @param response the collection of all responses that will have been read
      */
-    private static void verifyMocksCalled(DataRequestResponse response) {
+    private static void verifyMocksCalled(final DataRequestResponse response) {
 //        for (ConnectionDetail entry : response.getResources().values()) {
 //            verify(((DataService) entry.createService()), times(1)).read(any());
 //        }
     }
 
-    /**
-     * Creates a DataService that will respond to any request with the given data.
-     *
-     * @param dataToReturn collection of data that will be streamed back from the data reader
-     * @param shouldFail   if true the service will fail to return the data
-     * @return a mock data service instance
-     */
+//    /**
```

Comment:
Doesnt need both /*foo*/ commenting and // foo commenting. one or the other

Reply:
Removed the commented comments